administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -0,0 +1,166 @@
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+JHtml::addIncludePath(JPATH_COMPONENT . '/helpers/html');
+
+JHtml::_('bootstrap.tooltip');
+JHtml::_('behavior.multiselect');
+JHtml::_('formbehavior.chosen', 'select');
+JHtml::_('jquery.framework');
+
+$app    = JFactory::getApplication();
+$user   = JFactory::getUser();
+$userId = $user->get('id');
+$listOrder = $this->escape($this->state->get('list.ordering'));
+$listDirn  = $this->escape($this->state->get('list.direction'));
+
+$input = $app->input;
+
+$assoc = JLanguageAssociations::isEnabled();
+
+$componentFilter = $this->state->get('associationcomponent');
+
+if ($componentFilter != '')
+{
+	$parts     = explode('.', $componentFilter);
+	$comp      = $parts[0];
+	$assocItem = $parts[1];
+
+	JHtml::addIncludePath(JPATH_ADMINISTRATOR . '/components/' . $comp . '/helpers/html');
+
+	// Get the value in the Association column
+	if ($comp == "com_content")
+	{
+		$assocValue = "contentadministrator.association";
+	}
+	elseif ($comp == "com_categories")
+	{
+		$assocValue = "categoriesadministrator.association";
+	}
+	elseif ($comp == "com_menus")
+	{
+		$assocValue = "MenusHtml.Menus.association";
+	}
+	else
+	{
+		$assocValue = $assocItem . '.association';
+	}
+
+	// If it's not a category
+	if ($componentFilter != '' && !strpos($componentFilter, '|'))
+	{
+		$componentSplit = explode('.', $componentFilter);
+		$aComponent = $componentSplit[0];
+		$aView = $componentSplit[1];
+	}
+	elseif ($componentFilter != '') {
+		$componentSplit = explode('|', $componentFilter);
+		$aComponent = 'com_categories';
+		$aView = $componentSplit[1];
+	}
+
+	if (isset($aComponent) && isset($aView))
+	{
+		$link = 'index.php?option=com_associations&view=association&layout=edit&acomponent='
+		. $aComponent . '&aview=' . $aView . '&id=';
+	}
+}
+?>
+
+<?php if ($assoc) : ?>
+
+	<form action="<?php echo JRoute::_('index.php?option=com_associations&view=associations'); ?>" method="post" name="adminForm" id="adminForm">
+		<div id="j-main-container">
+
+		<?php echo JLayoutHelper::render('joomla.searchtools.default', array('view' => $this, 'options' => array('filterButton' => false))); ?>
+
+		<?php if (empty($this->items)) : ?>
+	        <div class="alert alert-no-items">
+	            <?php echo JText::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
+	        </div>
+	    <?php else : ?>
+			<table class="table table-striped" id="contactList">
+				<thead>
+					<tr>
+						<th class="nowrap">
+							<?php echo JHtml::_('searchtools.sort', 'JGLOBAL_TITLE', 'title', $listDirn, $listOrder); ?>
+						</th>
+						<th width="15%" class="nowrap hidden-phone">
+							<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_LANGUAGE', 'language_title', $listDirn, $listOrder); ?>
+						</th>
+						<?php if ($assoc) : ?>
+						<th width="5%" class="nowrap hidden-phone hidden-tablet">
```

Comment:
this is the main field of this view IMHO shoulnd't not be hidden in mobile/tablet view hidden-phone hidden-tablet